source/features/conversation-activity-filter.tsx

```diff
@@ -161,11 +161,11 @@ async function addWidget(state: State, anchor: HTMLElement): Promise<void> {
 			className={`details-reset details-overlay d-inline-block ml-2 position-relative ${dropdownClass}`}
 			id="rgh-conversation-activity-filter-select-menu"
 		>
-			<summary>
+			<summary className="height-full">
 				<EyeIcon className="color-fg-muted" />
```

Comment:
The class can probably be removed from here now

Reply:
What do you mean? Do you want it moved elsewhere?

Reply:
I meant it can probably be moved to its parent (and also removed from the caret). If not, the PR can be merged

Reply:
Ah sorry misled by the diff, didn't notice you were commenting about the color 😅 fixed!